documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -1371,14 +1378,23 @@ Connectors that capture changes from a sharded MongoDB cluster use this connecti
 After the initial discovery process, connection strings are generated for each individual shard.
 ====
 
+|[[mongodb-property-mongodb-connection-string-shard-params]]<<mongodb-property-mongodb-connection-string-shard-params, `+mongodb.connection.string.shard.params+`>>
+|No default
+|Specifies the URL parameters of https://www.mongodb.com/docs/manual/reference/connection-string/[connection string] that the connector uses to connect to individual shards of MongoDB sharded cluster.
```

Comment:
```suggestion
|Specifies the URL parameters of the https://www.mongodb.com/docs/manual/reference/connection-string/[connection string] that the connector uses to connect to individual shards of a MongoDB sharded cluster.
```
Does this property accept multiple strings? If so, I'd use the plural to reflect that:

>Specifies the URL parameters of the https://www.mongodb.com/docs/manual/reference/connection-string/[connection string**s**] that the connector uses to connect to individual shards of a MongoDB sharded cluster.`

Reply:
No it doesn't. It's a single query string. e.g. `readPreference=primary&sslEnabled=false`

Reply:
Although technically there are multiple connection strings  constructed with this parameter so I'm not quite sure whether plural or singular ought to be used here.  WDYT?